src/db/migrations/20190728221239-create-notification.js

```diff
@@ -0,0 +1,51 @@
+import Debug from 'debug';
+
+const debug = Debug('dev');
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    try {
+      await queryInterface.createTable('Notification', {
```

Comment:
```suggestion
      await queryInterface.createTable('Notifications', {
```
Please do the same thing to `down`

Reply:
done